sql-parser/src/main/java/io/crate/sql/SqlFormatter.java

```diff
@@ -340,6 +340,58 @@ public Void visitCreateUser(CreateUser node, Integer indent) {
         }
 
         @Override
+        public Void visitGrantPrivilege(GrantPrivilege node, Integer indent) {
+            builder.append("GRANT ");
+
+            EnumSet<Privilege> privileges = node.privileges();
+            int j = 0;
+            for (Privilege privilege : privileges ) {
+                builder.append(privilege.toString());
+                if (j < privileges.size() - 1) {
+                    builder.append(", ");
+                }
+                j++;
+            }
+
+            builder.append(" TO ");
+
+            List<String> userNames = node.userNames();
```

Comment:
Same here: Extract a method and reuse for revoke.
Or extract the whole visit code as a method and with an argument decide if you print GRANT or REVOKE and also FROM or TO